src/askscript/parser/askscript.grammar.pegjs

```diff
@@ -279,7 +281,7 @@ emptyLine = wsnonl* nl
 identifier = [_$a-zA-Z][_$a-zA-Z0-9]* { return new ask.Identifier(text()) } // TODO(lc): add Unicode here one day
 
 // operators consist of chars: -<>+*/^%=&|, but they cannot contain // sequence
-operator = ([-<>+*^%=&|] / ([/] &[^/]))+   {       return new ask.Identifier(text()) }
+operator = ([-<>+*^%=&|] / ([/] &[^/]))+   { return new ask.Identifier(text(), true) }
```

Comment:
We're going to save information in AST if identifier is an operator so that we can prettify those differently